pennylane/transforms/broadcast_expand.py

```diff
@@ -112,7 +112,12 @@ def broadcast_expand(tape):
 
     def processing_fn(results):
         if len(tape.measurements) > 1 and qml.active_return():
-            results = [qml.math.hstack(r) for r in results]
+            processed_results = [None] * len(tape.measurements)
+            for i in range(len(tape.measurements)):
+                processed_results[i] = qml.math.squeeze(
+                    qml.math.stack([results[b][i] for b in range(tape.batch_size)])
```

Comment:
did `broadcast_expand` not account for `batch_size` in the old return system, or did it happen auto-magically because of the shape of returned data?

Reply:
Second one, there was no need for processing.